charts/ckan/values.schema.json

```diff
@@ -63,6 +63,10 @@
           "type": "string",
           "description": "This sets the name of the site, as displayed in the CKAN web interface."
         },
+        "plugins":{
+          "type": "string",
```

Comment:
This should rather be an array of strings to be more user friendly

Reply:
you are right it is the nicer version. i have adapted it